CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+* Bump: Sentry-Cocoa to 7.16.0 (#886)
```

Comment:
Maybe wait for 7.17 which fixes https://github.com/getsentry/sentry-cocoa/issues/1877